docs/specification.md

```diff
@@ -166,12 +185,14 @@ Here's an example of a sequence collection organized into the canonical seqcol o
 This object would validate against the JSON Schema above.
 The object is a series of arrays with matching length (`3`), with the corresponding entries collated such that the first element of each array corresponds to the first element of each other array.
 For the rationale why this structure was chosen instead of an array of annotated sequences, see [*Footnote F1*](#f1-why-use-an-array-oriented-structure-instead-of-a-sequence-oriented-structure).
-Implementations `MUST` provide at least the structure specified in this schema.
+The implementation `MUST` define its structure in a JSON Schema, such as the example schema defined in step 1.
```

Comment:
Lines 188-190 now seems like a duplication of content in the new main section 1, so I think these should either be removed or merged into section 1 if one wants to retain the more "rule-focused" way this is written here. If a duplication of content is wanted, then it should refer to "section 1" and not "step 1".